compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1511,6 +1531,51 @@ static LogicalResult setRootConfig(func::FuncOp entryPointFn,
   return setConvRootConfig(entryPointFn, convOp, targetTileSizes, vectorSize);
 }
 
+static LogicalResult setRootConfig(func::FuncOp entryPointFn,
```

Comment:
As a bandaid, can we just have a common function to remove the duplicated lines. i.e create a `setConvRootConfigImpl` function that has the body of the function, and then each of these method just call that function.

Reply:
okay